cmd/minishift/cmd/root.go

```diff
@@ -59,6 +60,7 @@ const (
 	invalidProfileName    = "Profile names must consist of alphanumeric characters only."
 )
 
+var cmdIgnoreCreatingMinishiftHomeDir = []string{"version", "completion"}
```

Comment:
ignoreCreating?

Reply:
Ignore means "do not pay attention to" so, do not pay attention to me creating the homedirs?

preventHomeDirCreationForCommands = "version", "completion"

Reply:
For consistency, also write like:

```
var viperWhiteList = []string{
	"v",
	"alsologtostderr",
	"log_dir",
}
```

meaning

```
var preventHomeDirCreationForCommands = []string{
    "version",
    "completion",
}
```

Reply:
Works for me :+1: